.github/ISSUE_TEMPLATE/blank.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Open a blank issue.
+about: ''
+title: ''
+labels: triage
+assignees: ''
+---
+
+<!-- 
+Please don't post screenshots of error messages; 
+we can't help with issues like that.
+Please instead copy and paste any error messages.
+-->
```

Comment:
I'm still not comfortable about giving this option. I think people will see it as a shortcut to not having to fill out the issue template. Particularly when they see the *minimal example to reproduce the problem* and think *oh, but I've got a big closed source project. I can't fill that in*.

Reply:
Honestly, I'd expect in the occasional case where the issue template is inappropriate, the author will know to delete the template.

Reply:
But we've had this discussion before...

Reply:
> Honestly, I'd expect in the occasional case where the issue template is inappropriate, the author will know to delete the template.

The problem is that those have preassigned labels, `bug` or `feature`. I'd like to leave this for now, we can always remove it later easily enough.